audiophiler/static/js/audiophiler.js

```diff
@@ -1,15 +1,29 @@
 const toggleHarold = $(".toggleHarold");
 toggleHarold.click(function () {
     let hash = this.dataset.hash;
-    if(this.checked){
+    var isTour = "false"
+    if (tour.checked){
+        isTour = "true"
+    }
+
+    var params ={
+        tour: isTour
+    };
+
+    var json = JSON.stringify(params)
```

Comment:
This could definitely be less than 9 lines, since most of the variables defined here appear to never be used again.